ui/src/app/shared/action/action.component.ts

```diff
@@ -97,6 +114,26 @@ export class ActionComponent implements OnDestroy {
         }
     }
 
+    parseRequirements(): void {
+        // for each type 'model' and 'service', concat value with opts
+        // and replace \n with space
+        this.editableAction.requirements.map(req => {
```

Comment:
better to use `forEach` -> more explicit because we don't save the return value of map